hub/core/compression.py

```diff
@@ -349,7 +354,9 @@ def compress_multiple(
             b"".join(arr.tobytes() for arr in arrays), compression
         )  # Note: shape and dtype info not included
     elif compr_type == AUDIO_COMPRESSION:
-        raise NotImplementedError("compress_multiple does not support audio samples.")
+        raise NotImplementedError(
```

Comment:
Why is this under AUDIO_COMPRESSION

Reply:
I haven't changed anything, black changed only the way how it written. Take a look at this code snippet

[https://github.com/activeloopai/Hub/blob/main/hub/core/compression.py#:~:text=info%20not%20included-,elif%20compr_type%20%3D%3D%20AUDIO_COMPRESSION%3A,raise%20NotImplementedError(%22compress_multiple%20does%20not%20support%20audio%20samples.%22),-elif%20compr_type%20%3D%3D%20VIDEO_COMPRESSION]

Reply:
yes but you have changed "audio samples" to "cloud data samples"

Reply:
No, I haven't, could you please show explicitly where I have done that?
As far as I can see 
```

    elif compr_type == AUDIO_COMPRESSION:
        raise NotImplementedError("compress_multiple does not support audio samples.")
        raise NotImplementedError(
```
Nothing has changed

Reply:
Can you check line 358?